piman.py

```diff
@@ -106,6 +109,11 @@ def server():
     # when it is run, so after it is closed by a keyboard interrupt, the above
     # lines run closing the rest of the threads. ntp_thread will already be stopped
 
+# start monitoring thread
+def monitoring():
+    monitor_thread= Thread(target = monitoring_client.start_from_piman, name= "monitoring")
+    monitor_thread.start()
```

Comment:
if you are just going to do a start() then join(), you don't need a thread. you can just call it.